dev-requirements.txt

```diff
@@ -22,6 +22,9 @@ pluggy>=0.13.0
 colorama>=0.3.9,<0.4.0; python_version <= '3.4'
 colorama>=0.4.0; python_version >= '3.5'
 
+# more-itertools version 8.11 requires python 3.6, See issue #2796
+more-itertools<=8.10.0; python_version <= '3.5'
```

Comment:
Same comment as on pywbem:
- just exclude 8.11.0, but do not pin
- add 4.0.0 as a minimum.

Reply:
DONE